firmware/quintuna/CRIT/src/app/app_driveMode.c

```diff
@@ -0,0 +1,13 @@
+#include "app_driveMode.h"
+#include "app_canTx.h"
+#include "io_rotary.h"
+#include <stdint.h>
+
+#define NUM_MODES 7
+
+void app_driveMode_broadcast(void)
```

Comment:
i think there needs to be a bit more nuance here to handle the screen/other related activites on the 7segs